R/collectors.R

```diff
@@ -139,8 +140,9 @@ col_skip <- function() {
 #' @export
 #' @examples
 #' ## These all return 1000
-#' parse_number("$1,000") ## leading $ and grouping character , ignored
+#' parse_number("$1,000") ## leading $ and grouping character ignored
 #' parse_number("euro1,000") ## leading non-numeric euro ignored
+#' parse_number("t1000t1000") ## only parses first number found
```

Comment:
This example was pulled from the original issue https://github.com/tidyverse/readr/issues/1415